### ISSUE TITLE
Does this support Shadowsocks?

### ISSUE THREAD

**Issue by PowerPress** [rofl0r/proxychains-ng#463]
I want to use multi hop chain of shadowsocks servers, will this support it?

**Comment by rofl0r** [comment]
i dont know, but you can try it out and report your findings.

**Comment by Low-power** [comment]
No, because Shadowsocks protocol is not the original SOCKS protocol.
You must use a Shadowsocks client such as **ss-local(1)** (which itself is a SOCKS server) for use with proxychains.
